ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/default_interfaces.schema.yml

```diff
@@ -18,7 +18,10 @@ keys:
         types:
           type: list
           required: true
-          description: List of node type keys.
+          description: |
+            List of node type keys.
+            'default' will apply these default interfaces to node types where there is no more-specific match.
```

Comment:
this also means that you cannot use the node type "default" if you define your own node_type_keys without breaking this - probably best to mention it somewhere (like `default` is reserved or whatever in node_type_keys options) ? @ClausHolbechArista

Reply:
We should probably add a check for that.